mltsp/celery_tasks.py

```diff
@@ -28,7 +37,8 @@ def celery_available():
 
 
 @celery_app.task(name="celery_tasks.featurize_ts_file")
-def featurize_ts_file(ts_file_path, features_to_use, custom_script_path=None):
+def featurize_ts_file(ts_file_path, features_to_use, custom_script_path=None,
```

Comment:
This is such a thin wrapper--do we even need it?  Not for this PR, but we can think of taking it out in another.